src/client/runner/runner.js

```diff
@@ -83,13 +83,13 @@ Runner.prototype._onNextStepStarted = function (e) {
 Runner.prototype._onActionTargetWaitingStarted = function (e) {
```

Comment:
https://github.com/DevExpress/testcafe/pull/330/files#diff-619aff4845280917d61f0dfe5e1a6071R82
Replace with: 
//NOTE: decrease the step counter if the action restarts while waiting for an element to be available (T230851)